synfig-studio/src/gui/docks/dockmanager.cpp

```diff
@@ -420,6 +423,70 @@ bool DockManager::read_bool(std::string &x)
 	return read_string(x) == "true";
 }
 
+static void check_and_fix_window_position(int &x, int &y, int &width, int &height)
+{
+	// At least magic_mininum_length x magic_mininum_length pixels of window must be shown on display
+	const int magic_mininum_length = 50;
```

Comment:
constexpr?

Reply:
I don't get constexpr, but I believe you :)
Done